app/Http/Resources/MovieResource.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class MovieResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array|\Illuminate\Contracts\Support\Arrayable|\JsonSerializable
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
+            'title' => $this->title,
+            'year' => $this->year,
+            'description' => $this->description,
+            'user_id' => new UserResource($this->user),
```

Comment:
тут уже бужет объект пользователя)
так что не совсем id
```suggestion
            'user' => new UserResource($this->user),
```